### ISSUE TITLE
Sound wave animation

### ISSUE THREAD

**Issue by tormodAase** [equinor/attenuation-app#14]
https://www.figma.com/design/R5ZB5mHiO145XLrA7x7ife/Attenuation--%3E-Design?node-id=192-67794&t=cB6FUqOptwknEexg-1

@VetleW Do you know how to export this as an animation?

**Comment by tormodAase** [comment]
@dansvend Should we put this in the backlog? You could probably add some placeholder while Vetle is looking at this?

**Comment by dansvend** [comment]
> @dansvend Should we put this in the backlog? You could probably add some placeholder while Vetle is looking at this?

Yes, let's put it in the backlog and insert a placeholder for now 👌
